### PR TITLE
Fix broken link in README.md

### PR DESCRIPTION
**Pull request by pkirkovsky**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -62,7 +62,7 @@ Press `cmd+shift+r` (MacOS) / `ctrl+shift+r` (Windows) when you come across an e
 
 ### Download for [VS Code](https://marketplace.visualstudio.com/items?itemName=Continue.continue) and [JetBrains](https://plugins.jetbrains.com/plugin/22707-continue-extension)
 
-By default, Continue uses `GPT-4` and `GPT-3.5-turbo` via the OpenAI API. You can adjust the config to use different Large Language Models (LLMs) like [Code Llama, Claude 2, WizardCoder, PaLM 2, and more](https://github.com/continuedev/what-llm-to-use). Read more [here](https://continue.dev/docs/customization/models).
+By default, Continue uses `GPT-4` and `GPT-3.5-turbo` via the OpenAI API. You can adjust the config to use different Large Language Models (LLMs) like [Code Llama, Claude 2, WizardCoder, PaLM 2, and more](https://github.com/continuedev/what-llm-to-use). Read more [here](https://continue.dev/docs/customization/overview).
```
Comment (TyDunn) [inline README.md]:
```suggestion
By default, Continue uses `GPT-4` and `GPT-3.5-turbo` via the OpenAI API. You can adjust the config to use different Large Language Models (LLMs) like [Code Llama, Claude 2, WizardCoder, PaLM 2, and more](https://github.com/continuedev/what-llm-to-use). Read more [here](https://continue.dev/docs/model-setup/select-model).
```

**Review by TyDunn — APPROVED** [review]
Thanks!

**Comment by sestinj** [comment]
Hi @pkirkovsky, yesterday we shared [some updates](https://discordapp.com/channels/1108621136150929458/1131313996750917835/1371996789967687783) with our contributors about how we're aiming to improve the contribution process. Part of this included the addition of a [Contributor License Agreement](https://github.com/continuedev/continue/blob/main/docs/docs/CLA.md) (CLA) to protect both contributors and the project. We're reaching out to ask that previous contributors sign it.

Could you please take a moment to sign, or if you have any questions send me a message? (either here or nate@continue.dev would work)

To do so, you just need to **post a comment below with the following text:**<br/>
```
I have read the CLA Document and I hereby sign the CLA
```

❤️ Thank you for the work you've done on Continue, and let me know if you have any suggestions on how we can make the project even better!

**Comment by pkirkovsky** [comment]
I have read the CLA Document and I hereby sign the CLA

**Pull request merged**